vmdb/Gemfile

```diff
@@ -53,16 +53,18 @@ gem "ancestry",                       "~>1.2.4",      :require => false
 gem "aws-sdk",                        "~>1.56.0",     :require => false
 gem 'dalli',                          "~>2.2.1",      :require => false
 gem "elif",                           "=0.1.0",       :require => false
+gem "eventmachine",                   "=1.0.7",       :require => false
 gem "haml",                           "~>4.0.5",      :require => false
 gem 'haml-rails',                     "~> 0.4",       :require => false
 gem "inifile",                        "~>3.0",        :require => false
+gem "json",                           "=1.8.2",       :require => false
 gem "logging",                        "~>1.6.1",      :require => false  # Ziya depends on this
 gem "net-ping",                       "~>1.7.4",      :require => false
 gem "net-sftp",                       "~>2.0.5",      :require => false
 gem "net-ssh",                        "~>2.9.1",      :require => false
 gem "open4",                          "~>1.3.0",      :require => false
 gem "ovirt_metrics",                  "~>1.0.1",      :require => false
-gem "pg",                             "~>0.12.2",     :require => false
+gem "pg",                             "=0.12.2",      :require => false
```

Comment:
@jrafanie Using ~> as you suggest negates my intentions behind this PR. Perhaps changing the Gemfile is not the correct approach. I'll set up a meeting to discuss other options to pursue.